x-pack/qa/repository-old-versions-7x/build.gradle

```diff
@@ -0,0 +1,35 @@
+import org.elasticsearch.gradle.internal.test.RestIntegTestTask
+import org.elasticsearch.gradle.testclusters.StandaloneRestIntegTestTask
+import org.elasticsearch.gradle.Version
+
+apply plugin: 'elasticsearch.internal-java-rest-test'
+apply plugin: 'elasticsearch.rest-resources'
+
+restResources {
+  restApi {
+    include '_common', 'search'
+  }
+  restTests {
+    includeCore 'search/390_doc_values_search.yml'
```

Comment:
@breskeby next iteration: this yml test shows up now in the "build" directory but isn't executed on ":check"
Here's a local run: https://gradle-enterprise.elastic.co/s/cshioucnucegw
I copied over and modified the "DocValueOnlyFieldsIT" that should run this but I'm not sure how to wire it to the "check" task (or any other task for that matter). The setup might need adjustment still which I can do as long as the gradle wiring works. Thanks for your help.